tests/integration/testdata/invoke/cdk/cdk_template.yaml

```diff
@@ -1,73 +1,434 @@
 Resources:
-  StandardFunctionConstructImageFunction:
+  helloworldfunctionServiceRole306F1141:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+        Version: "2012-10-17"
+      ManagedPolicyArns:
+        - Fn::Join:
+            - ""
+            - - "arn:"
+              - Ref: AWS::Partition
+              - :iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+    Metadata:
+      aws:cdk:path: RootStack/helloworld-function/ServiceRole/Resource
+  helloworldfunctionFDE57595:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        S3Bucket:
+          Ref: AssetParametersfa3b13f8e96c4bf14d4c6a47337be97af700c05a39b382dd2b756f3f50ffd6b9S3Bucket0C6A52B4
+        S3Key:
+          Fn::Join:
+            - ""
+            - - Fn::Select:
+                  - 0
+                  - Fn::Split:
+                      - "||"
+                      - Ref: AssetParametersfa3b13f8e96c4bf14d4c6a47337be97af700c05a39b382dd2b756f3f50ffd6b9S3VersionKey5F29F473
+              - Fn::Select:
+                  - 1
+                  - Fn::Split:
+                      - "||"
+                      - Ref: AssetParametersfa3b13f8e96c4bf14d4c6a47337be97af700c05a39b382dd2b756f3f50ffd6b9S3VersionKey5F29F473
+      Role:
+        Fn::GetAtt:
+          - helloworldfunctionServiceRole306F1141
+          - Arn
+      Environment:
+        Variables:
+          MODE:
+            Ref: ModeEnvVariable
+      FunctionName: HelloWorldFunction
+      Handler: app.hello_world_handler
+      Runtime: python3.8
+    DependsOn:
+      - helloworldfunctionServiceRole306F1141
+    Metadata:
+      aws:cdk:path: RootStack/helloworld-function/Resource
+      aws:asset:path: ./lambda_code
+      aws:asset:original-path: ./
```

Comment:
original-path will be removed in the next CDK release, because of this issue https://github.com/aws/aws-cdk/issues/17706 .. anyway, we do not use it any where right now.

Reply:
Shouldn't make a difference, but I'll remove it to keep things up-to-date.